app/src/main/res/layout/concept_card_fragment.xml

```diff
@@ -58,12 +58,13 @@
             android:layout_marginEnd="@dimen/concept_card_heading_text_margin_end"
             android:fontFamily="sans-serif-medium"
             android:text="@{viewModel.conceptCardLiveData.conceptCard.skillDescription}"
-            android:textColor="@color/oppia_primary_text"
+            android:textColor="@color/component_color_concept_card_fragment_text_color"
             android:textSize="20sp" />
 
           <TextView
             android:id="@+id/concept_card_explanation_text"
             style="@style/Body"
+            android:textColor="@color/component_color_concept_card_fragment_text_color"
```

Comment:
Because of this line the static checks are failing.

Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
done @rt4914